pkg/controllers/user/controllers.go

```diff
@@ -70,6 +70,7 @@ func Register(ctx context.Context, cluster *config.UserContext, clusterRec *mana
 	istio.Register(ctx, cluster)
 	certsexpiration.Register(ctx, cluster)
 	ingresshostgen.Register(ctx, cluster.UserOnlyContext())
+	ingress.Register(ctx, cluster.UserOnlyContext())
```

Comment:
Why is this getting moved up to management?

Reply:
We need access to settings, which is not available in user context

Reply:
Ingress controller was moved to the user context to address scalability issues. If we could find a way to propagate the setting to the user cluster (storing it in ingress annotations perhaps?), that would be great

Reply:
So did this always just use xip.io then? It was using settings before, so perhaps an unseen bug?

Reply:
No, we added a check in this controller which requires reading the ip domain from the settings.